### DISCUSSION TITLE
Spawn/create new tab with wezterm cli

### DISCUSSION THREAD

**Discussion by weiying-chen** [wezterm/wezterm#5046] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Wayland/Mutter

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

With wezterm cli, you can move a pane to a new tab with this: `wezterm cli move-pane-to-new-tab`.

But can you create a new tab? Like when you press CTRL + SHIFT + T?

**Comment by wez** [comment, marked as answer]
https://wezfurlong.org/wezterm/cli/cli/spawn.html
